Header header
string[] names
SplineTrajectorySegment[] segments
